src/__tests__/RenderParentDashboard.test.js

```diff
@@ -40,7 +40,7 @@ describe('<ParentDashboard /> test suite', () => {
     expect(screen.getByText(/Welcome back/i)).toBeInTheDocument();
   });
 
-  test('Change user Nav', () => {
+  test.skip('Change user Nav', () => {
     render(<Component parent={{ children: [] }} />);
 
```

Comment:
After consultation with Sara - these tests (which are testing the old parent dashboard) are skipped to allow a merge.